src/Handler/Workspace/Drop/DropWorkspaceHandler.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\Teradata\Handler\Workspace\Drop;
+
+use Google\Protobuf\Internal\Message;
+use Keboola\StorageDriver\Command\Workspace\DropWorkspaceCommand;
+use Keboola\StorageDriver\Contract\Driver\Command\DriverCommandHandlerInterface;
+use Keboola\StorageDriver\Credentials\GenericBackendCredentials;
+use Keboola\StorageDriver\Teradata\TeradataSessionManager;
+use Keboola\TableBackendUtils\Escaping\Teradata\TeradataQuote;
+use Throwable;
+
+final class DropWorkspaceHandler implements DriverCommandHandlerInterface
+{
+    private TeradataSessionManager $manager;
+
+    public function __construct(TeradataSessionManager $manager)
+    {
+        $this->manager = $manager;
+    }
+
+    /**
+     * @inheritDoc
+     * @param GenericBackendCredentials $credentials
+     * @param DropWorkspaceCommand $command
+     */
+    public function __invoke(
+        Message $credentials, // project credentials
+        Message $command,
+        array $features
+    ): ?Message {
+        assert($credentials instanceof GenericBackendCredentials);
+        assert($command instanceof DropWorkspaceCommand);
+
+        // validate
+        assert(!empty($command->getWorkspaceUserName()), 'DropWorkspaceCommand.workspaceUserName is required');
+        assert(!empty($command->getWorkspaceRoleName()), 'DropWorkspaceCommand.workspaceRoleName is required');
+        assert(!empty($command->getWorkspaceObjectName()), 'DropWorkspaceCommand.workspaceObjectName is required');
+
+        $db = $this->manager->createSession($credentials);
+
+        try {
+            $db->executeStatement(sprintf(
+                'DROP ROLE %s;',
+                TeradataQuote::quoteSingleIdentifier($command->getWorkspaceRoleName())
+            ));
+
+            $db->executeStatement(sprintf(
+                'DROP USER %s;',
+                TeradataQuote::quoteSingleIdentifier($command->getWorkspaceUserName())
+            ));
+
+            $db->executeStatement(sprintf(
+                'DROP DATABASE %s',
+                TeradataQuote::quoteSingleIdentifier($command->getWorkspaceObjectName()),
+            ));
```

Comment:
Tady to imho nebude fungovat pokud tam bude nějaký objekt budeš muset ještě zavolat `DELETE DATABASE %s ALL`

Reply:
nemelo by tam byt teda i `isCascade` jako u bucketu?
```php
        if ($command->getIsCascade() === true) {
            try {
                $db->executeStatement(sprintf(
                    'DELETE DATABASE %s ALL',
                    TeradataQuote::quoteSingleIdentifier($command->getBucketObjectName())
                ));
             ...
         }

        $db->executeStatement(sprintf(
            'DROP DATABASE %s',
            TeradataQuote::quoteSingleIdentifier($command->getBucketObjectName()),
        ));
```

Reply:
pridano `isCascade`